From c58fd33a0a172a460ff6196cde6e84877b204243 Mon Sep 17 00:00:00 2001
From: Fei Jiang <fei.jiang@intel.com>
Date: Tue, 12 Jun 2018 10:42:16 +0800
Subject: [PATCH 1576/2345] drm/i915/gvt: fix a problem to set wrong register
 PLANE_WM

Fixed problem in commit "drm/i915/gvt: local display support",
need write register PLANE_WM_TRANS instead of PLANE_WM

Signed-off-by: Fei Jiang <fei.jiang@intel.com>
Reviewed-by: Min He <min.he@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/intel_pm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index a5b0075..22169a2 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -4832,7 +4832,7 @@ static void skl_write_plane_wm(struct intel_crtc *intel_crtc,
 				&wm->trans_wm);
 	}
 #else
-	skl_write_wm_level(dev_priv, PLANE_WM(pipe, plane_id, level),
+	skl_write_wm_level(dev_priv, PLANE_WM_TRANS(pipe, plane_id),
 			&wm->trans_wm);
 #endif
 
-- 
2.7.4

